actionview/lib/action_view/helpers/tags/base.rb

```diff
@@ -33,29 +33,29 @@ def render
 
         private
 
-          def value(object)
+          def value
             if @allow_method_names_outside_object
-              object.public_send @method_name if object && object.respond_to?(@method_name)
+              @object.public_send @method_name if @object && @object.respond_to?(@method_name)
```

Comment:
Changing `object` to `@object` is unnecessary because the Base class has `attr_reader :object`.